packages/opentelemetry-plugin-fetch/src/enums/AttributeNames.ts

```diff
@@ -28,4 +28,5 @@ export enum AttributeNames {
   HTTP_URL = 'http.url',
   HTTP_TARGET = 'http.target',
   HTTP_USER_AGENT = 'http.user_agent',
+  TRANSFER_SIZE = 'transferSize',
```

Comment:
I think the name `transferSize` is incorrect with regards to semantic conventions
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#common-attributes
and spec for transfer size https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/transferSize

It should be probably `http.response_content_length_uncompressed` @dyladan @mayurkale22 WDYT ?.
If that is not the case we should come out with some name proposition maybe then `http.response_size`

Reply:
I think `http.response_content_length_uncompressed` is probably correct in this case.

Reply:
Actually, I think uncompressed would be `PerformanceResourceTiming.decodedBodySize` and `PerformanceResourceTiming.transferSize` is `http.response_content_length`

Reply:
OK, so looking at the specs, I think that `http.response_content_length` most closely matches `encodedBodySize`; nothing seems to match `transferSize`.  According to caniuse.com these two fields have equivalent adoption in browsers, so perhaps I should use `encodedBodySize`->`http.response_content_length`?

Reply:
That works for me. I would also add `decodedBodySize` => `http.response_content_length_uncompressed` iff `encodedBodySize !== decodedBodySize`

Reply:
I left off the decoded size for now; I'll circle back to it later.